packages/i18n/src/locales/en.i18n.json

```
@@ -2220,6 +2220,7 @@
   "error-registration-not-found": "Registration information not found",
   "error-extension-not-available": "Extension not available",
   "error-user-not-found": "User not found",
+  "error-voip-disaled": "VoIP for Team Collaboration is disabled",
```

Might be a good idea to change this to be the same as the settings group.
```suggestion
  "error-voip-disaled": "Team voice calls (VoIP) is disabled",
```

---

I had copied "VoIP for Team Collaboration" from the settings page title... Should it be replaced there as well?